File: cpp/program/AnggotaDPR.cpp
#include <iostream>
#include <string>

using namespace std;

class AnggotaDPR
{
    private:
        int id;
        string nama;
        int umur;
        string jabatan;
        string namaBidang;
        string namaPartai;

    public:
        /* Constructors. */
        AnggotaDPR() {
            this->id = 0;
            this->nama = "";
            this->umur = 0;
            this->jabatan = "";
            this->namaBidang = "";
            this->namaPartai = "";
        }
    
        AnggotaDPR(int id, string nama, int umur, string jabatan, string namaBidang, string namaPartai) {
            this->id = id;
            this->nama = nama;
            this->umur = umur;
            this->jabatan = jabatan;
            this->namaBidang = namaBidang;
            this->namaPartai = namaPartai;
        }

        int get_id() const {
            return id;
        }

        string get_nama() const {
            return nama;
        }

        int get_umur() const {
            return umur;
        }

        string get_jabatan() const {
            return jabatan;
        }

        string get_namaBidang() const {
            return namaBidang;
        }

        string get_namaPartai() const {
            return namaPartai;
        }

        void set_id(int newId) {
            id = newId;
        }

        void set_nama(string newNama) {
            nama = newNama;
        }

        void set_umur(int newUmur) {
            umur = newUmur;
        }

        void set_jabatan(string newJabatan) {
            jabatan = newJabatan;
        }

        void set_namaBidang(string newNamaBidang) {
            namaBidang = newNamaBidang;
        }

        void set_namaPartai(string newNamaPartai) {
            namaPartai = newNamaPartai;
        }

        void show_identitas() const {
            cout << "ID: " << id << ", Nama: " << nama << ", Umur: " << umur
                << ", Jabatan: " << jabatan << ", Bidang: " << namaBidang
                << ", Partai: " << namaPartai << endl;
        }

        /* Destructors. */
        ~AnggotaDPR() {

        }
};

File: python/program/AnggotaDPR.py
class AnggotaDPR:
    def __init__(self, id=0, nama="", umur=0, jabatan="", namaBidang="", namaPartai=""):
        self.__id = id
        self.__nama = nama
        self.__umur = umur
        self.__jabatan = jabatan
        self.__namaBidang = namaBidang
        self.__namaPartai = namaPartai

    def get_id(self):
        return self.__id

    def get_nama(self):
        return self.__nama

    def get_umur(self):
        return self.__umur

    def get_jabatan(self):
        return self.__jabatan

    def get_namaBidang(self):
        return self.__namaBidang

    def get_namaPartai(self):
        return self.__namaPartai

    def set_id(self, id):
        self.__id = id

    def set_nama(self, nama):
        self.__nama = nama

    def set_umur(self, umur):
        self.__umur = umur

    def set_jabatan(self, jabatan):
        self.__jabatan = jabatan

    def set_namaBidang(self, namaBidang):
        self.__namaBidang = namaBidang

    def set_namaPartai(self, namaPartai):
        self.__namaPartai = namaPartai

    def show_identitas(self):
        print(f"ID: {self.__id}, Nama: {self.__nama}, Umur: {self.__umur}, "
              f"Jabatan: {self.__jabatan}, Bidang: {self.__namaBidang}, "
              f"Partai: {self.__namaPartai}")
